src/Estimators/EstimatorManagerInput.cpp

```diff
@@ -50,23 +65,23 @@ void EstimatorManagerInput::readXML(xmlNodePtr cur)
       if (aname.empty() && !atype.empty())
         aname = atype;
       if (atype == "localenergy" || atype == "elocal")
-        appendScalarEstimatorInput<LocalEnergyInput>(child);
+        scalar_estimator_inputs_.emplace_back(std::in_place_type<LocalEnergyInput>, child);
```

Comment:
I think this is just a style thing.  No different from any other function that just improves the reading of the code.

Reply:
Likely. Probably will change appendScalarEstimatorInput to a lambda instead of a member function.

Reply:
Restored and simplified. FYI template lambda is C++20.